### PR TITLE
Enable file processing debug output

### PR DESCRIPTION
**Pull request by dblandin**
When the `--debug` flag is passed, output file path before processing payload.

This should provide valuable debug information when dealing with
encoding errors or other file content issues.

```bash
$ codeclimate-test-reporter --debug
Processing file path=codeclimate_test_reporter/__init__.py
Processing file path=codeclimate_test_reporter/__main__.py
Processing file path=codeclimate_test_reporter/components/__init__.py
Processing file path=codeclimate_test_reporter/components/api_client.py
Processing file path=codeclimate_test_reporter/components/argument_parser.py
Processing file path=codeclimate_test_reporter/components/ci.py
Processing file path=codeclimate_test_reporter/components/file_coverage.py
Processing file path=codeclimate_test_reporter/components/formatter.py
Processing file path=codeclimate_test_reporter/components/git_command.py
Processing file path=codeclimate_test_reporter/components/payload_validator.py
Processing file path=codeclimate_test_reporter/components/reporter.py
Processing file path=codeclimate_test_reporter/components/runner.py
```

@codeclimate/review :mag_right:

### CODE REVIEW

File: codeclimate_test_reporter/components/runner.py [hunk 1]
```diff
@@ -38,11 +38,11 @@ def run(self):
         """
 
         try:
-            if self.parsed_args.debug:
-                self.out.write(self.__debug_info())
-                return 0
-            elif self.parsed_args.version:
-                self.out.write(reporter_version)
+            if self.parsed_args.version:
+                if self.parsed_args.debug:
```
Comment (pbrisbin) [inline codeclimate_test_reporter/components/runner.py]: Why is this now dependent on `--version`?

Comment (dblandin) [inline codeclimate_test_reporter/components/runner.py]: I guess it doesn't have to be. Main change is to not exit right away and continue processing. I'll move this back up and just remove the exit.

**Comment by dblandin** [comment]
@pbrisbin Ready for another :mag_right:

**Comment by pbrisbin** [comment]
LGTM

**Pull request merged**